docs/apm/apm-alerts.asciidoc

```diff
@@ -0,0 +1,99 @@
+[role="xpack"]
+[[apm-alerts]]
+=== Create an alert
+
+beta::[]
+
+The APM app is integrated with Kibana's {kibana-ref}/alerting-getting-started.html[alerting and actions] feature.
+It provides a set of built-in **actions** and APM specific threshold **alerts** for you to use,
+and allows all alerts to be centrally managed from <<management,Kibana Management>>.
+
+[role="screenshot"]
+image::apm/images/apm-alert.png[Create an alert in the APM app]
+
+There are two different types of threshold alerts: transaction duration, and error rate.
+Below, we'll create one of each.
+
+[float]
+[[apm-create-transaction-alert]]
+=== Create a transaction duration alert
+
+This guide creates an alert for the `opbeans-java` service based on the following criteria:
+
+* Transaction type: `transaction.type:request`
+* Average request is above `1500ms` for the last 5 minutes
+* Check every 10 minutes, and repeat the alert every 30 minutes
+* Send the alert via Slack
+
+From the APM app, navigate to the `opbeans-java` service and select
+**Alerts** > **Create threshold alert** > **Transaction duration**.
+
+The name of your alert will automatically be set as `Transaction duration | opbeans-java`,
+and the alert will be tagged with `apm` and `service.name:opbeans-java`.
+Feel free to edit either of these defaults.
+
+Based on the alert criteria, define the following alert details:
+
+* **Check every** - `10 minutes`
+* **Notify every** - `30 minutes`
+* **TYPE** - `request`
+* **WHEN** - `avg`
+* **IS ABOVE** - `1500ms`
+* **FOR THE LAST** - `5 minutes`
+
+Select an action type.
+Multiple action types can be selected, but in this example we want to post to a slack channel.
+Select **Slack** > **Create a connector**.
+Enter a name for the connector,
+and paste the webhook URL.
+See Slack's webhook documentation if you need to create one.
+
+Select **Save**. The alert has been created and is now active!
+
+[float]
+[[apm-create-error-alert]]
+=== Create an error rate alert
+
+This guide creates an alert for the `opbeans-python` service based on the following criteria:
+
+* Transaction type: `transaction.type:celery`
```

Comment:
An error rate alert does not take `transaction.type` as a parameter.

Reply:
Ahh you've found lots of good copy/paste mistakes. Thanks